zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -314,6 +318,9 @@ public String getKeyManagerPassword() {
 
   public String getTrustStorePath() {
     String path = getString(ConfVars.ZEPPELIN_SSL_TRUSTSTORE_PATH);
+    if (path == null) {
+      path = getKeyStorePassword();
```

Comment:
Shouldn't this be `getKeyStorePath()`? And that should mean that if keystore is used as truststore then `getTrustStorePassword()` need the same logic